src/components/shared/LanguageSelector/LanguageSelector.js

```diff
@@ -1,41 +1,51 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { injectIntl } from 'react-intl';
+import { InjectAppServices } from '../../../services/pure-di';
 
 /**
  * LanguageSelector
  * @param { Object } props
  * @param { import('react-intl').InjectedIntl } props.intl
  */
-const LanguageSelector = ({ intl }) => {
+const LanguageSelector = ({
+  intl,
+  dependencies: {
+    window: { Menubutton },
+  },
+}) => {
   const lang = intl.locale;
+  const languageDropdown = useRef(null);
+
+  useEffect(() => {
+    const menubutton = new Menubutton(languageDropdown.current);
+    menubutton.init();
+  }, [languageDropdown]);
+
   return (
-    <div className="language-selector">
-      {lang === 'es' ? (
-        <>
-          <div className="lang-option option--selector">
-            <button className="lang--es">ES</button>
-          </div>
-          <div className="lang-option">
-            <Link to="?lang=en" className="lang--en">
-              EN
-            </Link>
-          </div>
-        </>
-      ) : (
-        <>
-          <div className="lang-option option--selector">
-            <button className="lang--en">EN</button>
-          </div>
-          <div className="lang-option">
-            <Link to="?lang=es" className="lang--es">
-              ES
-            </Link>
-          </div>
-        </>
-      )}
+    <div className="dp-c-dropdown language-selector">
+      <button
+        className={'lang--' + lang}
+        id="menubutton"
+        aria-haspopup="true"
+        aria-controls="menu2"
```

Comment:
the aria attr it's use in UI library?

Reply:
It is, @fcoronelmakingsense , or at least we are using it in components that need some user interaction, such as this one.